scripts/dropstat

```diff
@@ -457,39 +482,22 @@ class DropStat(object):
         else:
             return PORT_STATE_NA
 
-
-def main():
-    parser = argparse.ArgumentParser(description='Display drop counters',
-                                     formatter_class=argparse.RawTextHelpFormatter,
-                                     epilog="""
-Examples:
-  dropstat
-""")
-
-    # Version
-    parser.add_argument('-v', '--version', action='version', version='%(prog)s 1.0')
-
-    # Actions
-    parser.add_argument('-c', '--command', type=str, help='Desired action to perform')
-
-    # Variables
-    parser.add_argument('-g', '--group',   type=str, help='The group of the target drop counter', default=None)
-    parser.add_argument('-t', '--type',    type=str, help='The type of the target drop counter', default=None)
-
-    args = parser.parse_args()
-
-    command = args.command
-
-    group = args.group
-    counter_type = args.type
-
-    dcstat = DropStat()
+@click.command(help='Display drop counters')
```

Comment:
why do we need to add this change here?
`dropstat` is standalone python script. the click options need to present in `show/dropcounter.py`

Reply:
The original script also supported argument parsing. Using the `click` simplified that logic significantly. It was suggested earlier by Wenyi and I agree this is an improvement over what was originally implemented. Furthermore, the UTs directly call this script.